test/check_energy_and_forces.py

```diff
@@ -0,0 +1,82 @@
+import pickle
+import os
+import numpy as np
+import sys
+import multiprocessing as mp
+import argparse
+from ase.io import read, Trajectory
+from tqdm import tqdm
+
+def check_relaxed_forces(sid, path, thres):
+    """
+    Check all forces in the final frame of adslab is less than a threshold.
+    """
+    final_atoms = read(path)
+    forces = final_atoms.get_forces()
+    if not (np.max(np.abs(forces)) <= thres):
+        print(f"{sid} doesn't satisfy the force threshold")
+
+def check_adsorption_energy(sid, path, ref_energy, adsorption_energy):
+    final_energy = read(path)
+    if not abs((final_energy.get_potential_energy() - ref_energy) - adsorption_energy) < 1e-6:
+        print(f"{sid} doesn't satify energy equation")
+
+def check_DFT_energy(sid, path):
+    """
+    Given a relaxation trajectory, check to see if 1. final energy is less than the initial
+    energy, raise error if not. 2) If energies monotonically decrease in a trajectory.
```

Comment:
Nit: I wouldn't call it "monotonically decreasing" if a small increase of < 0.05 is ok. Also, is 0.05 always an appropriate threshold, or should it be an arg?

Reply:
added as an arg, updated wording